package.json

```diff
@@ -131,7 +131,6 @@
     "sinon": "^11.1.2",
     "sinon-chai": "^3.7.0",
     "tap": "^16.3.4",
-    "tape": "^4.9.1",
-    "wait-on": "^5.0.0"
+    "tape": "^4.9.1"
```

Comment:
Why do we depend on both tap _and_ tape? 🤔

Reply:
That's unrelated to this PR, but it's because there used to be memory leak tests that used `tape`. These tests were problematic and haven't run for a long time, we might consider removing them. But again, that's out of the scope of this PR.